app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    parts = cp.split(" ")
+    old = parts[1]
+    new = parts[2]
```

Comment:
you can assign variables into one line

`command, old, new = input.split(" ")`

Also old and new is a bad names, maybe try smth. like source and destination>?